app/helpers/repository_helper.rb

```diff
@@ -1,14 +1,26 @@
 module RepositoryHelper
-  def periodicity_options
-    [["Not Periodically", 0], ["1 day", 1], ["2 days", 2], ["Weekly", 7], ["Biweekly", 15], ["Monthly", 30]]
+  def periodicity_options(language)
+    current_locale = "#{I18n.locale}"
+    portuguese_regex = "pt"
+
+    if(language == portuguese_regex)
+      portuguese_periodicity_options
+    else
+      [["Not Periodically", 0], ["1 day", 1], ["2 days", 2], ["Weekly", 7], ["Biweekly", 15], ["Monthly", 30]]
+    end
+  end
+
+  def portuguese_periodicity_options
+    [["Sem periodicidade", 0], ["1 dia", 1], ["2 dias", 2], ["Semanal", 7], ["Quinzenal", 15], ["Mensal", 30]]
```

Comment:
What do you think about creating those into I18n translation keys? So you don't need to use implement this hardcoded method for pt and this translation gets generic for any future locale.